pkg/collect/host_filesystem_performance_linux.go

```diff
@@ -50,46 +36,36 @@ func collectHostFilesystemPerformance(hostCollector *troubleshootv1beta2.Filesys
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
-	var operationSize uint64 = 1024
-	if hostCollector.OperationSizeBytes != 0 {
-		operationSize = hostCollector.OperationSizeBytes
-	}
-
-	var fileSize uint64 = 10 * 1024 * 1024
-	if hostCollector.FileSize != "" {
-		quantity, err := resource.ParseQuantity(hostCollector.FileSize)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to parse fileSize %q", hostCollector.FileSize)
-		}
-		fileSizeInt64, ok := quantity.AsInt64()
-		if !ok {
-			return nil, errors.Wrapf(err, "failed to parse fileSize %q", hostCollector.FileSize)
-		}
-		fileSize = uint64(fileSizeInt64)
+	collectorName := hostCollector.CollectorName
+	if collectorName == "" {
+		collectorName = "filesystemPerformance"
 	}
+	name := filepath.Join("host-collectors/filesystemPerformance", collectorName+".json")
 
-	if hostCollector.Directory == "" {
-		return nil, errors.New("Directory is required to collect filesystem performance info")
-	}
-	// TODO: clean up this directory if its created
 	if err := os.MkdirAll(hostCollector.Directory, 0700); err != nil {
 		return nil, errors.Wrapf(err, "failed to mkdir %q", hostCollector.Directory)
 	}
-	filename := filepath.Join(hostCollector.Directory, "fsperf")
 
-	f, err := os.OpenFile(filename, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0600)
-	if err != nil {
-		log.Panic(err)
-		return nil, errors.Wrapf(err, "open %s", filename)
-	}
-	defer func() {
-		if err := f.Close(); err != nil {
-			log.Println(err.Error())
-		}
-		if err := os.Remove(filename); err != nil {
-			log.Println(err.Error())
-		}
-	}()
+	// filename := filepath.Join(hostCollector.Directory, fioJobOptions.Name)
+	// // Create file handle
```

Comment:
@adamancini were we to clean this up?

Reply:
I thought I had removed this before pushing the last commit 😠